data-prepper-core/src/main/java/org/opensearch/dataprepper/acknowledgements/InactiveAcknowledgementSetManager.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.acknowledgements;
+
+import org.opensearch.dataprepper.model.event.Event;
+import org.opensearch.dataprepper.model.event.EventHandle;
+import org.opensearch.dataprepper.model.acknowledgements.AcknowledgementSetManager;
+import org.opensearch.dataprepper.model.acknowledgements.AcknowledgementSet;
+import java.util.function.Consumer;
+import java.time.Duration;
+
+public class InactiveAcknowledgementSetManager implements AcknowledgementSetManager {
+    private static InactiveAcknowledgementSetManager theInstance;
+
+    public static InactiveAcknowledgementSetManager getInstance() {
+        if (theInstance == null) {
+            theInstance = new InactiveAcknowledgementSetManager();
+        }
+        return theInstance;
+    }
+    
+    public AcknowledgementSet create(final Consumer<Boolean> callback, final Duration timeout) {
+        throw new UnsupportedOperationException("create operation not supported");
```

Comment:
Do we want to throw an exception here or just ignore and log the call?

Reply:
We should throw exception because these operations should not be invoked on it.